python/scripts/bump.py

```diff
@@ -23,16 +23,16 @@ class BumpType(Enum):
 
 def _get_bumped_version(current: VersionInfo, btype: BumpType) -> VersionInfo:
```

Comment:
Consider adding a docstring to the `_get_bumped_version` function to explain the change in API from `bump_*` to `next_version`. This would help future maintainers understand why this change was made and what edge cases it addresses.